### ISSUE TITLE
Consumer rejects emails with Re: or Fwd: in subject

### ISSUE THREAD

**Issue by Whisprin** [the-paperless-project/paperless#607]
I'm importing documents from an email inbox. However, email subjects containing "Fwd:" or "Re:" are considered unsafe and the email is not processed further.

The code for checking the subject is here:
https://github.com/the-paperless-project/paperless/blob/2cd077d12d1f3046a02b9ed96f3acb4ac30e25cd/src/documents/mail.py#L108

The subject line is matched against SAFE_REGEX: https://github.com/the-paperless-project/paperless/blob/1c956652f360e58409c8fca148b7662585dd1087/src/documents/models.py#L160

If there are no objections I'm going to draft a PR to make unsafe subjects safe and then continue with the import. A similar suggestion has been made in: https://github.com/the-paperless-project/paperless/issues/506

**Comment by ahyear** [comment]
This might be problematic if you are spamed for any reason.
I would advise against it.
Why don't you change the subject before sending it?

**Comment by ddddavidmartin** [comment]
Sounds good to me.  I don't see how colons or other punctuation make emails more unsafe than not having them in there.

Also, isn't the whole point of the `SHARED_SECRET` to filter out emails that are not intended for paperless? The subject line shouldn't really matter as long as the secret is contained in the email I would think.

```
# If you intend to consume documents either via HTTP POST or by email, you must
# have a shared secret here.
PAPERLESS_SHARED_SECRET="some passphrase that is unlikely to occur in random emails"
```

**Comment by Whisprin** [comment]
I agree that the spam issue is already solved by `PAPERLESS_EMAIL_SECRET`.

First, I would strip a set of common subject prefixes like "Fwd:" and then remove all non-safe characters from the subject.

**Comment by totti4ever** [comment]
I would highly appreciate this! Would also second that the current situation is by no means a spam protection or similar

**Comment by fabean** [comment]
Same, I'm trying to get emails from my scanner to go in but their subject is `Transfer file from "Epson Connect Scan to Cloud"`

It'd also be neat if it could not care and just take an entire email account, I setup a whole new email address for this purpose that all documents go to. If I cannot get this to work, I'll probably use OfflineIMAP to sync in all emails into the consume directory.
